media-proxy/src/shm_memif_rdma.c

```diff
@@ -113,10 +113,10 @@ int tx_rdma_on_disconnect(memif_conn_handle_t conn, void *priv_data)
     }
 
     // release session
-    if (!tx_ctx->shm_ready) {
+    if (!atomic_load(&tx_ctx->shm_ready)) {
         return 0;
     }
-    tx_ctx->shm_ready = 0;
+    atomic_store(&tx_ctx->shm_ready, false);
```

Comment:
```suggestion
    atomic_store_explicit(&tx_ctx->shm_ready, false, memory_order_release);
```

Reply:
Again. Here I used relaxed instead of release. We don't modify any memory before this store operation, so there is no need to barrier.